Mathlib/Tactic/Ring/PNat.lean

```diff
@@ -19,12 +19,13 @@ instance : CSLift ℕ+ Nat where
   lift := PNat.val
   inj := PNat.coe_injective
 
--- FIXME: this `no_index` seems to be in the wrong place, but
+-- FIXME: this `ofNat` seems to be in the wrong place, but
 -- #synth CSLiftVal (3 : ℕ+) _ doesn't work otherwise
-instance {n} : CSLiftVal (no_index (OfNat.ofNat (n+1)) : ℕ+) (n + 1) := ⟨rfl⟩
```

Comment:
I think I'd leave this as it was for now, so as not to mangle the already ambiguous comment.

Reply:
The weird thing is that the `#synth` command seems to work whether we have the `no_index` or not. So I agree, let's not touch this yet.